packages/react-component-library/src/components/RangeSlider/ThresholdTrack.tsx

```diff
@@ -15,13 +15,34 @@ export interface ThresholdTrackProps extends TrackItem {
 }
 
 export interface ChunkProps {
+  getTrackProps: GetTrackProps
   $left: number
   $width: number
   $maxWidth: number
   testId: string
   $thresholdColor?: ThresholdColor
 }
 
+const Chunk: React.FC<ChunkProps> = ({
```

Comment:
Although this change is fine for `RangeSliderE` it causes a weird rendering bug in the original `RangeSlider` implementation.


![2022-01-24 11 01 54](https://user-images.githubusercontent.com/48086589/150771290-67f7b377-4cb2-4aa7-ba07-0db9f1ff298f.gif)

Reply:
Have pushed a fix for that: https://5e25c277526d380020b5e418-chdlredmtk.chromatic.com/?path=/docs/range-slider--double-threshold

(Although this is targetting `release/3.0.0` anyway :smile: )